src/xcode/ENA/ENA/Resources/Localization/en.lproj/privacy-policy.html

```diff
@@ -744,10 +744,7 @@ <h2>
 			and organisational details of this cooperation are laid down in an EU
 			Commission Decision (Commission Implementing Decision (EU) 2020/1023 of
 			15 July 2020, which is available at
-			<a href="https://eur-lex.europa.eu/eli/dec_impl/2020/1023/oj">
-				https://eur-lex.europa.eu/eli/dec_impl/2020/1023/oj</a>
-			 
-			).
+			<a href="https://eur-lex.europa.eu/eli/dec_impl/2020/1023/oj">		https://eur-lex.europa.eu/eli/dec_impl/2020/1023/oj</a>).
```

Comment:
change in English version ?

Reply:
removed an obsolete whitespace character.